### DISCUSSION TITLE
How to add custom properties in user profile in API list

### DISCUSSION THREAD

**Discussion by ttkrpink** [mediacms-io/mediacms#457] [Q&A]
I have changed the model file, and added one field in the user profile. Then, I changed the form.py file, and added one field in the form so a user can change the property in the UI's "Edit Profile" page.

My question is: How to add the field in the API's 'User Lists' property.

Another question is, How to return user's list with filter, something like "is_manager"=true

Thanks for your help.

**Comment by ttkrpink** [comment]
Find it in serializers.py file.

**Reply by mgogoulos** [reply]
Hi, when you change models, you have to run `python manage.py makemigrations` to produce the DB migrations file and then `python manage.py migrate` to apply it, I'm sure you did but mentioning it here just in case!

users/views.py and users/serializers.py should be enough for  this, unless you are addin a new API endpoint, in this case you need to add it to users.urls.py
You can see examples there that GET HTTP parameters are checked if they are passed through the API, and if so there are filteres performed to the querysets that return data. 

Cheers
